custom_components/ha_creality_ws/camera.py

```
@@ -850,26 +1009,304 @@ async def async_handle_async_webrtc_offer(
+                out.append("a=group:BUNDLE " + " ".join(mids))
+                replaced = True
+            else:
+                out.append(line)
+        if not replaced and mids:
+            out.append("a=group:BUNDLE " + " ".join(mids))
+        return out
+
+    def _rejected_media_answer(self, offer_section: list[str], mid: str) -> list[str]:
+        """Return a rejected answer media section preserving the offered kind."""
+        if not offer_section:
+            return [f"m=application 0 UDP/DTLS/SCTP webrtc-datachannel", f"a=mid:{mid}", "a=inactive"]
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)